### DISCUSSION TITLE
How to start contributing a component to paste?

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#2103] [Office hours] [answered]
Roland has been working on a toggle component and has the link in story book - is this something we could start actively contributing to the paste system? How?

Links to mockups or prototype: [Storybook link here](https://console-storybook.netlify.app/?path=/story/applications-verify-shared-toggle--with-help-text)

**Comment by loreina** [comment, marked as answer]
Hi @mjanssen2021, this is so great to hear! We don't have any docs for component contributions just yet but the process is fairly similar to contributing a pattern: https://paste.twilio.design/patterns/contribute/

I'll reach out over Slack to discuss details. Excited for this! 🙌
